### PR TITLE
Create Spanish localization for event-driven-architecture.md

### LINKED ISSUE #1373: [Spanish] localize 'Event-Driven Architecture'
Localize the `content/en/event-driven-architecture.md` document to Spanish

### PR DESCRIPTION
**Pull request by anairinac**
Signed-off-by: Irina Calvo <anairinac@gmail.com>

### Describe your changes

Created the Spanish localization for the event-driven-architecture.md document

### Related issue number or link (ex: `resolves #issue-number`)

Resolves #1373

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/event-driven-architecture.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Arquitectura Basada en Eventos
+status: Feedback Appreciated
```
Comment (electrocucaracha) [inline content/es/event-driven-architecture.md:3]:
```suggestion
status: Completed
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]
Solo algunos cambios menores en la traducción. Por cierto es necesario agregar `HTTP` y `Kafka` en el word list.

```bash
$ echo "HTTP" >> content/es/.wordlist.txt
$ echo "Kafka" >> content/es/.wordlist.txt
$ sort -f -o content/es/.wordlist.txt content/es/.wordlist.txt
```

File: content/es/event-driven-architecture.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+category: Concepto
+tags: ["", "", ""]
```
[the remark this answers is not in the document]
Comment (krol3) [reply content/es/event-driven-architecture.md:5]: @anairinac como estas? necesitas alguna ayuda para aplicar los comentarios de @electrocucaracha ?

[the remark this answers is not in the document]
Comment (anairinac) [reply content/es/event-driven-architecture.md:5]: Hola! Esta noche voy a aplicar todos los cambios, no he podido porque he estado trabajando mucho 😳

**Review by electrocucaracha — APPROVED** [review]

**Review by ramrodo — APPROVED** [review]
/lgtm

**Comment by electrocucaracha** [comment]
@raelga @92nqb @krol3 can you merge this PR?

**Review by raelga — APPROVED** [review]

**Pull request merged**